### ISSUE TITLE
[Bug]: OidcSecurityService.logoff() AuthOptions.redirectUrl is ignored

### ISSUE THREAD

**Issue by Pascalmh** [damienbod/angular-auth-oidc-client#1506]
**What Version of the library are you using?**
14.1.3

**Describe the bug**
When calling the `logoff()`-Method with the `redirectUrl` set via the 2nd param (AuthOptions) the information is lost and the default logoff url (postLogoutRedirectUri) is used.

**To Reproduce**

Call the logoff method like this:
```
const redirectUrl = 'https://some-url.com/some-route';
this.oidcSecurityService.logoff(undefined, { redirectUrl });
```
The `OidcSecurityService.logoff`-method calls the `LogoffRevocationService.logoff`-method, which just ignores the redirectUrl:

```
// https://github.com/damienbod/angular-auth-oidc-client/blob/release_14_1_3/projects/angular-auth-oidc-client/src/lib/logoff-revoke/logoff-revocation.service.ts#L30
const { urlHandler, customParams } = authOptions || {};
```

**Expected behavior**
The User gets redirected to the url set as `redirectUrl`.

**Comment by damienbod** [comment]
released in v15.0.0

**Comment by NiclasMolby** [comment]
@damienbod v.15 introduced the `LogoutAuthOptions` which does not contain an option to override `post_logout_redirect_uri`
How can this be accomplished now?
